### ISSUE TITLE
Exception option for individual mods from REAWheels

### ISSUE THREAD

**Issue by DonPaul75** [900hasse/FS22_REAwheels#16]
Hello, excellent work. One question. It would be possible to insert an exception option from this mode. For example, for motorcycles and bicycles, for these, realism is not so important in FS. but with REAWheel it is not possible to use these norms. For example, it would be enough the possibility of writing to the vehicle XML that it should be ignored for the REAWheel. So far, many thanks for the excellent work.

**Comment by DrewCZ** [comment]
I have the same experience, but caused by REAcog - check here for the same issue: https://github.com/900hasse/FS22_REAcog/issues/3

**Comment by DonPaul75** [comment]
> I have the same experience, but caused by REAcog - check here for the same issue: [900hasse/FS22_REAcog#3](https://github.com/900hasse/FS22_REAcog/issues/3)

Yes, it's the same  problem with all other REA mods. Similar notation for the exception from the "exhaustExtension" mod : 		exhaustExtension excludeVehicle="false/true"

 900hasse has just to set this option in his mod and then it will be prefect.

**Comment by Vanquish081** [comment]
Same poblem with the Pivots in Land of La Mancha.
